src/resolvers/Query/getAgendaCategoryById.ts

```diff
@@ -0,0 +1,40 @@
+import type { QueryResolvers } from "../../types/generatedGraphQLTypes";
+import { AgendaCategoryModel } from "../../models";
+import { errors, requestContext } from "../../libraries";
+import { AGENDA_CATEGORY_NOT_FOUND_ERROR } from "../../constants";
+/* eslint-disable */
```

Comment:
Why this comment?

Reply:
Removed all the eslint disable comments.